src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -15,8 +17,10 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniquePersonList persons;
+    private final UniqueIngredientList ingredients;
```

Comment:
IngredientList should not be under AddressBook, as AddressBook should only be in charge of contact details.